include/zephyr/drivers/display.h

```diff
@@ -46,6 +46,7 @@ enum display_pixel_format {
 	PIXEL_FORMAT_ARGB_8888		= BIT(3), /**< 32-bit ARGB */
 	PIXEL_FORMAT_RGB_565		= BIT(4), /**< 16-bit RGB */
 	PIXEL_FORMAT_BGR_565		= BIT(5), /**< 16-bit BGR */
+	PIXEL_FORMAT_L_8		= BIT(6), /**< 8-bit Grayscale/Luminance */
```

Comment:
How about a comment to list the other formats like "also known as 'GRAY', 'GRAY8', 'Y8'"

Reply:
Done